core/views.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+import jwt
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models import User
+from freepass  import settings
+
+def signin_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            access_token = request.headers.get('Authorization')
+            payload = jwt.decode(access_token, settings.SECRET_KEY, algorithms=settings.ALGORITHM)
+            user = User.objects.get(kakao_id=payload['user_id'])
+            request.user = user
+
+        except jwt.exceptions.DecodeError:
+            return JsonResponse({'message' : 'INVALID_TOKEN'}, status=400)
+        except User.DoesNotExist:
+            return JsonResponse({'message' : 'INVALID_TOKEN'}, settings=400)
+        
+        return func(self, request, *args, **kwargs)
```

Comment:
1. return은 except보다 위에 위치시켜 주세요.
2. return이 들여쓰기가 안 되어 있는 것 같습니다.
```suggestion
            return func(self, request, *args, **kwargs)
        
        except jwt.exceptions.DecodeError:
            return JsonResponse({'message' : 'INVALID_TOKEN'}, status=401)
        except User.DoesNotExist:
            return JsonResponse({'message' : 'INVALID_TOKEN'}, settings=401)
```